pkg/generate/code/set_sdk_test.go

```diff
@@ -1778,3 +1778,28 @@ func TestSetSDK_MQ_Broker_Create(t *testing.T) {
 		code.SetSDK(crd.Config(), crd, model.OpTypeCreate, "r.ko", "res", 1),
 	)
 }
+
+func TestSetSDK_EC2_VPC_ReadMany(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+
+	g := testutil.NewModelForService(t, "ec2")
+
+	crd := testutil.GetCRDByName(t, g, "Vpc")
+	require.NotNil(crd)
+
+	expected := `
+	if r.ko.Spec.DryRun != nil {
+		res.SetDryRun(*r.ko.Spec.DryRun)
+	}
+	if r.ko.Status.VPCID != nil {
+		f4 := []*string{}
+		f4 = append(f4, r.ko.Status.VPCID)
+		res.SetVpcIds(f4)
+	}
```

Comment:
This is 100% correct. Just pointing that If an array/map is made of Go built-in type we could just do:

```go
res.SetVpcIds(r.ko.Status.VPCID)
```

Reply:
Ah good point. If this is preferred by the team, then I could just call `setSDKForScalar` instead of the emptyVar call, then the append, then the `setSDKForScalar`. 

**Any opinions/preferences from the team?**

Reply:
I prefer to keep it the way it is, since it is the way it is due to the need to construct both scalar and complex Go types for nested fields. It's a little verbose, but it's consistent for all types, not just builtin types.